codegen/java-gen/src/main/scala/com/lightbend/akkasls/codegen/java/EntityServiceSourceGenerator.scala

```diff
@@ -206,75 +207,59 @@ object EntityServiceSourceGenerator {
             method(
               "public",
               qualifiedType(state.fqn),
-              "snapshot",
-              List.empty,
-              emptyDoc
-            ) {
-              "// TODO: produce state snapshot here" <> line <>
-              "return" <+> qualifiedType(
-                state.fqn
-              ) <> dot <> "newBuilder().build()" <> semi
-            } <> line <>
-            line <>
-            "@Override" <>
-            line <>
-            method(
-              "public",
-              "void",
-              "handleSnapshot",
-              List(
-                qualifiedType(state.fqn) <+> "snapshot"
-              ),
-              emptyDoc
-            ) {
-              "// TODO: restore state from snapshot here" <> line
-            } <> line <> line
-          case _ => emptyDoc
-        }) <>
-        ssep(
-          service.commands.toSeq.map { command =>
-            "@Override" <>
-            line <>
-            method(
-              "public",
-              "Reply" <> angles(qualifiedType(command.outputType)),
-              lowerFirst(command.fqn.name),
-              List(
-                qualifiedType(command.inputType) <+> "command",
-                text("CommandContext")
-                <+> "ctx"
-              ),
+              "emptyState",
+              Nil,
               emptyDoc
-            ) {
-              "return Reply.failure" <> parens(notImplementedError("command", command.fqn)) <> semi
-            }
-          },
-          line <> line
-        ) <>
-        (entity match {
-          case ModelBuilder.EventSourcedEntity(_, _, _, events) =>
+            )(
+              "return" <+> qualifiedType(state.fqn) <> ".getDefaultInstance();"
+            ) <>
             line <>
             line <>
             ssep(
-              events.toSeq.map { event =>
+              service.commands.toSeq.map { command =>
                 "@Override" <>
                 line <>
                 method(
                   "public",
-                  "void",
-                  lowerFirst(event.fqn.name),
+                  "Effect" <> angles(qualifiedType(command.outputType)),
+                  lowerFirst(command.fqn.name),
                   List(
-                    qualifiedType(event.fqn) <+> "event"
+                    qualifiedType(state.fqn) <+> "currentState",
+                    qualifiedType(command.inputType) <+> "command"
                   ),
                   emptyDoc
                 ) {
-                  "throw new RuntimeException" <> parens(
-                    notImplementedError("event", event.fqn)
-                  ) <> semi
+                  "return effects().error" <> parens(notImplementedError("command", command.fqn)) <> semi
```

Comment:
same here, command name instead of "command". Though, not clear what "command" means here.

Reply:
Should have been event I think, copy pasta.